lib/coinbase/stake.rb

```diff
@@ -0,0 +1,191 @@
+# frozen_string_literal: true
+
+require 'faraday'
+
+module Coinbase
+  # A representation of a Stake operation used for a end-user custody flow.
+  class Stake
+    class << self
+      VALID_PROTOCOL_NETWORK = [
+        'ethereum-holesky'
+      ].freeze
+
+      SELF_SERVICE_INTEGRATOR_CONTRACT_ADDRESS_HOLESKY = '0xA55416de5DE61A0AC1aa8970a280E04388B1dE4b'
```

Comment:
This can be removed. The backend always defaults to a meaningful contract address.

Reply:
So leave the integrator contract address blank you're saying? in the post body?

Reply:
Just don't pass it :) Is that not possible ?

Reply:
Yeah got it, done!